app/scripts/lib/fxa-client.js

```diff
@@ -363,6 +357,13 @@ function (_, FxaClient, $, xhr, p, Session, AuthErrors, Constants) {
         });
     },
 
+    accountKeys: function (keyFetchToken, unwrapBKey) {
+      return this._getClient()
+              .then(function (client) {
```

Comment:
nit: indentation. We started off with this formatting, then went back to the standard 2 space indent for `.then`:

```js
  return this._getClient()
    .then(function (client) {
```